[2020-10-22 00:17:29,553] INFO [GroupCoordinator 0]: Member kafka-python-2.0.2-df093d73-9c6b-4445-b982-d09dfd08f9ec in group my-group has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2020-10-22 00:17:29,579] INFO [GroupCoordinator 0]: Preparing to rebalance group my-group in state PreparingRebalance with old generation 1 (__consumer_offsets-12) (reason: removing member kafka-python-2.0.2-df093d73-9c6b-4445-b982-d09dfd08f9ec on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2020-10-22 00:17:30,479] INFO [GroupCoordinator 0]: Group my-group with generation 2 is now empty (__consumer_offsets-12) (kafka.coordinator.group.GroupCoordinator)
[2020-10-22 00:17:31,320] WARN Client session timed out, have not heard from server in 3804523ms for sessionid 0x100167948f40005 (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:32,327] INFO Client session timed out, have not heard from server in 3804523ms for sessionid 0x100167948f40005, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:33,938] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:33,940] INFO Socket connection established, initiating session, client: /127.0.0.1:61298, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:33,946] WARN Unable to reconnect to ZooKeeper service, session 0x100167948f40005 has expired (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:33,946] INFO EventThread shut down for session: 0x100167948f40005 (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:33,948] INFO Unable to reconnect to ZooKeeper service, session 0x100167948f40005 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:33,947] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2020-10-22 00:17:33,956] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2020-10-22 00:17:33,957] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=18000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@7334aada (org.apache.zookeeper.ZooKeeper)
[2020-10-22 00:17:33,962] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2020-10-22 00:17:33,963] INFO zookeeper.request.timeout value is 0. feature enabled= (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:33,970] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:33,972] INFO Socket connection established, initiating session, client: /127.0.0.1:61301, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:33,985] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2020-10-22 00:17:34,153] INFO Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x100167948f40006, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2020-10-22 00:17:34,334] INFO Stat of the created znode at /brokers/ids/0 is: 257,257,1603307854155,1603307854155,1,0,0,72082304208732166,200,0,257
 (kafka.zk.KafkaZkClient)
[2020-10-22 00:17:34,335] INFO Registered broker 0 at path /brokers/ids/0 with addresses: PLAINTEXT://DESKTOP-3DEOF7P:9092, czxid (broker epoch): 257 (kafka.zk.KafkaZkClient)
